tests/dragonfly/server_family_test.py

```diff
@@ -66,17 +64,21 @@ def test_connection_name(client):
     name = client.execute_command("CLIENT GETNAME")
     assert name == "test_conn_name"
 
+
 '''
 make sure that the scan command is working with python
 '''
+
+
 def test_scan(client):
-    try:
-        for key, val in gen_test_data(n=10, seed="set-test-key"):
-            res = client.set(key, val)
-            assert res is not None
-            cur, keys = client.scan(cursor=0, match=key, count=2)
-            assert cur == 0
-            assert len(keys) == 1
-            assert keys[0] == key
-    except Exception as e:
-        assert False, str(e)
+    def gen_test_data():
+        for i in range(10):
+            yield "key-"+str(i), "value-"+str(i)
```

Comment:
nitpick - yield f"key-{i}" f"value-{i}"
BTW why did you removed the "gen_test_data"?

Reply:
Because its now the only place where it would be used... Maybe I should keep it though